index.ts

```diff
@@ -149,20 +148,14 @@ const fetchOmnivore = async (
         }
 
         if (highlights?.length) {
-          const highlightBlock = await logseq.Editor.insertBlock(
-            articleBlock.uuid,
-            highlightTitle
-          )
-          if (!highlightBlock) {
-            throw new Error('block error')
-          }
-
-          for (const highlight of highlights) {
-            await logseq.Editor.insertBlock(
-              highlightBlock.uuid,
-              highlight.quote
-            )
-          }
+          const highlightBatch = highlights.map(it => {
+            return {
+              content: `${it.quote} - [read](https://omnivore.app/me/${slug}#${it.id})`
+            }
+          })
+          await logseq.Editor.insertBatchBlock(articleBlock.uuid, highlightBatch, {
```

Comment:
does the `insertBatchBlock` api work now?

Reply:
yeah! I downloaded the reddit sample app to try and debug it, and noticed it just worked. Not sure if its because `sibling` needed to be set or something else.